src/components/tidspunkt/DatoOgKlokkeslett.tsx

```diff
@@ -14,21 +14,21 @@ const DatoOgKlokkeslett: React.FC<{tidspunkt: string; bareDato?: boolean; brukKo
         !(bareDato && bareDato === true) && new Date(tidspunkt).getHours() + new Date(tidspunkt).getMinutes() > 0;
     return (
         <>
-            <span className="dato">
+            <time className="dato">
```

Comment:
Stilig, visste ikke om denne. Kult at den passer med React-intl sin FormattedDate.
Er fortsatt litt forvirra for hva den gjør. Den gjør kanskje ikke så mye med utseende men er nyttig pga det gjør det lettere for maskiner å forstå hva teksten innebærer?
`allowing for better search engine results or custom features such as reminders.`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
Ja, den gjør nok ikke så mye her. Men generelt er det et bra prinsipp å bruke spesifikke html-tager der det er mulig (istedet for div'er overalt), gjør det også enklere for skjermlesere og lese opp og navigere i tekst!